js/tools/markerBrush.js

```diff
@@ -0,0 +1,33 @@
+window.tools.markerBrush = {
```

Comment:
this file seems almost identical to the example `pencil.js` file, did u mean to include this in ur PR? is this something u're still working on?